decidim-admin/app/forms/decidim/admin/organization_appearance_form.rb

```diff
@@ -47,7 +45,6 @@ class OrganizationAppearanceForm < Form
       translatable_attribute :omnipresent_banner_title, String
       translatable_attribute :omnipresent_banner_short_description, String
 
-      validates :cta_button_path, format: { with: %r{\A[a-zA-Z]+[a-zA-Z0-9\-\_/]+\z} }, allow_blank: true
```

Comment:
I would move it to the `ContentBlockForm` and add a validation that will be bypassed if the content_block's `manifest_name` is not :hero.
While the number of editable content blocks is manageable I would stay with a single form object. When we have many it would be great to use a factory method to get a different form object for each block.